### PR TITLE
Add md comments to prs

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
+    branches: [ add-md-comments ]
```
Comment (rom1504) [inline .github/workflows/ci.yml:7]: wrong

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -42,12 +42,12 @@ jobs:
         java-package: jre
     - run: npm install
     - name: Use Dumper
-      run: |
-        mkdir output
-        node bin/dumpPackets -v ${{ matrix.mcVersionIndex }} -o output/${{ matrix.mcVersionIndex }}
-        
+      run: node bin/dumpPackets -v ${{ matrix.mcVersionIndex }}
```
Comment (rom1504) [inline .github/workflows/ci.yml:45]: this is incorrect, why did you remove the `output/${{ matrix.mcVersionIndex }}` ?

Comment (u9g) [reply .github/workflows/ci.yml]: must've merged wrong

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -42,12 +42,12 @@ jobs:
...
+    - name: Save PR number
+      run: echo ${{ github.event.number }} > ./output/NR
```
Comment (rom1504) [inline .github/workflows/ci.yml:47]: this is going to write several times to the same file, putting it in build could do the trick (you can have another upload artifact pushing to the same artifact name there)

File: .github/workflows/publish.yml [hunk 4]
```diff
@@ -1,25 +1,48 @@
-name: npm-publish
+# Done this way because of https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+name: Comment on the pull request
```
Comment (rom1504) [inline .github/workflows/publish.yml:2]: why did you change this file ?

Comment (rom1504) [reply .github/workflows/publish.yml:2]: ah I see

publish.yml already existed, use another file name

File: .github/workflows/make-comment.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
+# Done this way because of https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+name: Comment on the pull request
+
+# read-write repo token
+# access to secrets
+on:
+  workflow_run:
+    workflows: ["ci"]
```
Comment (rom1504) [inline .github/workflows/make-comment.yml:8]: it's CI

**Review by rom1504 — APPROVED** [review]
